crates/nu-command/src/dataframe/series/date/get_weekday.rs

```diff
@@ -35,7 +35,7 @@ impl Command for GetWeekDay {
             result: Some(
                 NuDataFrame::try_from_columns(vec![Column::new(
                     "0".to_string(),
-                    vec![Value::test_int(1), Value::test_int(1)],
+                    vec![Value::test_int(2), Value::test_int(2)],
```

Comment:
I've checked calendar, and found that 2020-08-04 is Tuesday, not sure why it passed before :-(
```
❯ cal --full-year 2020 -m | where month == 8 | reject month
╭───┬────────┬────────┬─────────┬───────────┬──────────┬────────┬──────────╮
│ # │ sunday │ monday │ tuesday │ wednesday │ thursday │ friday │ saturday │
├───┼────────┼────────┼─────────┼───────────┼──────────┼────────┼──────────┤
│ 0 │        │        │         │           │          │        │        1 │
│ 1 │      2 │      3 │       4 │         5 │        6 │      7 │        8 │
│ 2 │      9 │     10 │      11 │        12 │       13 │     14 │       15 │
│ 3 │     16 │     17 │      18 │        19 │       20 │     21 │       22 │
│ 4 │     23 │     24 │      25 │        26 │       27 │     28 │       29 │
│ 5 │     30 │     31 │         │           │          │        │          │
╰───┴────────┴────────┴─────────┴───────────┴──────────┴────────┴──────────╯
```

Reply:
Thats kind of weird that it passed before. Glad it's fixed now.